app/build.gradle

```diff
@@ -32,10 +32,16 @@ android {
 
 dependencies {
     implementation project(":libndt7")
-    implementation "com.android.support:appcompat-v7:$support_library_version"
+    //implementation "com.android.support:appcompat-v7:$support_library_version"
     implementation "com.android.support.constraint:constraint-layout:$constraint_layout_version"
     implementation "com.squareup.okhttp3:okhttp:$okhttp_version"
     implementation "org.jetbrains.kotlin:kotlin-stdlib-jdk7:$kotlin_version"
+    implementation "com.google.code.gson:gson:$gson_version"
+    implementation "com.jakewharton:butterknife:$butterknife_version"
+    implementation "com.google.android.material:material:+"
```

Comment:
What's the meaning of `+`?

Reply:
Generally, it's used to retrieve the last version of that lib